### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by mkckr0** [mkckr0/audio-share#88] [Announcements]
- `as-cmd` can now specify the encoding, channels and sample rate. But on Windows it has some limitations. For more details, see the `About Audio Format` in README.
- Add "Encoding" option and `Sound Panel` button for server MFC.
- Add the default LAN address selection for `as-cmd`.

<hr /><em>This discussion was created from the release <a href='https://github.com/mkckr0/audio-share/releases/tag/v0.2.0'>v0.2.0</a>.</em>

**Comment by outusuke** [comment]
linux version i'm getting Segmentation fault (core dumped)

**Reply by mkckr0** [reply]
Can you post the core dump file?

**Reply by outusuke** [reply]
```
Module libpipewire-0.3.so.0 from rpm pipewire-1.2.6-1.fc41.x86_64
                Stack trace of thread 86231:
                #0  0x000056266ab10314 n/a (as-cmd + 0x58314)
                #1  0x000056266ab103ea n/a (as-cmd + 0x583ea)
                #2  0x000056266ab09a39 n/a (as-cmd + 0x51a39)
                #3  0x00007f440b480248 __libc_start_call_main (libc.so.6 + 0x3248)
                #4  0x00007f440b48030b __libc_start_main@@GLIBC_2.34 (libc.so.6 + 0x330b)
                #5  0x000056266ab0d635 n/a (as-cmd + 0x55635)
                ELF object binary architecture: AMD x86-64
```
                
the segfault occurs specifically when the VPN  is active

**Reply by mkckr0** [reply]
You may have some problem with #93
